lib/sparql-queries.ts

```diff
@@ -122,85 +118,10 @@ export async function executeDeleteQuery (quads: RDF.Quad[]) {
   }
 }
 
-export async function getLatestTimestamp (baseResource: RDF.NamedNode, treeProperties: TreeProperties) {
-  const quads = [
-    quad(variable("version"), treeProperties.versionOfPath, baseResource),
-    quad(variable("version"), treeProperties.timestampPath, variable("timestamp"))
-  ];
-  const variables = [variable("timestamp")];
-  const sparqlQuery = constructSelectQuery(variables, quads, 'ORDER BY DESC(?timestamp)');
-  try {
-    const response = await query(sparqlQuery);
-    const timestamp = extractVariableFromResponse(response, "timestamp")?.shift();
-    if (timestamp) {
-      return timestamp;
-    }
-    return;
-  } catch (e) {
-    console.error(e);
-  }
-}
-
 export async function executeDeleteInsertQuery (
   quadsToDelete: RDF.Quad[],
   quadsToInsert: RDF.Quad[]
 ) {
   await executeDeleteQuery(quadsToDelete);
   await executeInsertQuery(quadsToInsert);
-}
-
-export async function fetchState (stream: RDF.NamedNode): Promise<State | undefined> {
```

Comment:
The ldes-client stores its state as a file on disk. Although we could write and read this data to the triplestore, it makes working with the client more cumbersome. I decided to leave it out here, instead users of this service can provide a volume mount to store the state of ldes-client.